packages/ui/components/input-datepicker/test/lion-input-datepicker.test.js

```diff
@@ -410,6 +411,31 @@ describe('<lion-input-datepicker>', () => {
         expect(el.__calendarMaxDate.toString()).to.equal(new Date('2019/07/15').toString());
       });
 
+      it('should update the central date of Calendar if updated validator made the current central date disabled', async () => {
+        const initialValidators = [new MaxDate(new Date('2000/01/01'))];
+        const updatedValidators = [new MaxDate(new Date('2020/01/01'))];
+
+        const el = await fixture(html`
+          <lion-input-datepicker .validators="${initialValidators}"></lion-input-datepicker>
+        `);
+        el._overlayInvokerNode.click();
+        await new Promise(res => {
```

Comment:
Humm, I just tested this but didn't work.

And I think I know why. Basically, we already wait for the calendarNode is updated before we call `initCentralDate()`, because `initCentralDate()` is called through `_onCalendarOverlayOpened()` which is called in `__openCalendarOverlay()` after we wait for `this._calendarNode.updateComplete`: https://github.com/ing-bank/lion/blob/3d1c723d39bb1df448489c57675245616a9d75ae/packages/ui/components/input-datepicker/src/LionInputDatepicker.js#L326-L343

Reply:
Alright thanks for checking. Luckily we had the ["page object"](https://github.com/ing-bank/lion/blob/3d1c723d39bb1df448489c57675245616a9d75ae/packages/ui/components/input-datepicker/test-helpers/DatepickerInputObject.js) fixing this :)